reports/report_builder.py

```diff
@@ -3128,8 +3128,34 @@ def ws_103(self, ws):
         self.tabulate(ws, counts, YESNO, MAJOR_TOPICS, row_perc=True) 
         
     def ws_104(self, ws):
-        return
-    
+        """
+        Cols: Function in story
+        Rows: Major topic, covid stories only, Sex of source
+        """
+        r = 6
+        self.write_col_headings(ws, FUNCTION)
+
+        for major_topic, topic_ids in GROUP_TOPICS_MAP.items():
+            counts = Counter()
+            for media_type, model in person_models.items():
+                # some Person models don't have a function field
+                if 'function' in [field_name.name for field_name in model._meta.get_fields()]:
+                    rows = model.objects \
+                            .values('sex', 'function') \
+                            .filter(**{model.sheet_name() + '__covid19': 1}) \
+                            .filter(**{model.sheet_name() + '__country__in': self.country_list}) \
+                            .filter(**{model.sheet_name() + '__topic__in': topic_ids}) \
+                            .filter(sex__in=self.male_female_ids) \
```

Comment:
I think we are missing the `Do not know ` and `Other (transgender, etc.) ` Gender. 
I think we should be using `.filter(sex__in=[x[0] for x in GENDER])`

Reply:
I thought about and went back to the report; most of the sheets just show Female and Male so maybe we should change the [columns](https://github.com/CodeForAfrica/gmmp/pull/149/files#diff-19a686dbe88bb9f2f8c4f2ce5e331aa6f6db260926a2a5b7f1edfa7864a649b4R3156) to just `self.male_female` instead of `GENDER`?

Reply:
I think doing this might cause us to lose some data. 
![Screenshot 2021-03-04 at 08 21 48](https://user-images.githubusercontent.com/13068580/109915343-b2c9c780-7cc2-11eb-81c2-5c14197f67fd.png)

Reply:
Yeah, I think lets include all gender and request for confirmation for Jean/Sarah